src/pages/home/sidebar/OptionRow.js

```diff
@@ -163,22 +164,29 @@ const OptionRow = (props) => {
                             {
                                 !_.isEmpty(props.option.icons)
                                 && (
-                                    <MultipleAvatars
-                                        avatarImageURLs={props.option.icons}
-                                        size={props.mode === 'compact' ? 'small' : 'default'}
-                                        secondAvatarStyle={[
-                                            StyleUtils.getBackgroundAndBorderStyle(props.backgroundColor),
-                                            props.optionIsFocused
-                                                ? StyleUtils.getBackgroundAndBorderStyle(focusedBackgroundColor)
-                                                : undefined,
-                                            hovered && !props.optionIsFocused
-                                                ? StyleUtils.getBackgroundAndBorderStyle(hoveredBackgroundColor)
-                                                : undefined,
-                                        ]}
-                                        isChatRoom={props.option.isChatRoom}
-                                        isArchivedRoom={props.option.isArchivedRoom}
-                                        avatarTooltips={avatarTooltips}
-                                    />
+                                    props.option.shouldShowSubscript ? (
+                                        <SubscriptAvatar
```

Comment:
Have you tested this in #focus mode? Just looking at how `MultipleAvatars` is used in this component, and it looks like there was a need to adjust the size of the avatar if `props.mode === 'compact'`

Reply:
Ohh I missed that, checking that. Also, I'm kind of confused about the #focus mode, this only changes the size, right? because noticed in the production app that chats disappear from LHN and only the Chronos chat is shown

Reply:
> Also, I'm kind of confused about the #focus mode, this only changes the size, right? because noticed in the production app that chats disappear from LHN and only the Chronos chat is shown

Yeah, there's more that changes with #focus mode. I think it only shows chats for which you have unread messages